tests/baselines/reference/docker/vscode.log

```diff
@@ -1,11 +1,24 @@
-Exit Code: 0
+Exit Code: 1
```

Comment:
this is worse than before, right? Is this due to our test harness changing or vscode changing?

Reply:
We haven't been building `vscode` for awhile, so there's now way to know. This is what `vscode` reports for `master` *now* (after getting the build to actually run in #42431).

Reply:
This is probably something we can ignore? The error reported is that `monaco.d.ts` changed (from [here](https://github.com/microsoft/vscode/blob/a095d7fcf74d0b072886c3041bd5932f3b5be445/build/lib/compilation.js#L171)), which is likely due to Anders's type alias naming PR. The error will likely go away once vscode is building using 4.2 and generates a stable copy of monaco.d.ts.